pom.xml

```diff
@@ -14,7 +14,7 @@
 
     <properties>
         <!-- Baseline Jenkins version you use to build the plugin. Users must have this version or newer to run. -->
-        <jenkins.version>2.73.1</jenkins.version>
+        <jenkins.version>2.73.3</jenkins.version>
         <!-- Java Level to use. Java 7 required when using core >= 1.612 -->
         <java.level>7</java.level>
```

Comment:
Why not 8?

Reply:
dunno, likely it was just left as a default

Reply:
Yes, I think for master we could upgrade to 8 but note that we have a branch 1.x which is compatible with older Jenkins versions. I don't know if I'll maintain 1.x anymore due to lack of time.

Reply:
Also, issue #17 explains why the base line is so high.